sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -2193,15 +2211,119 @@ object SupportedOpsForTools {
     }
   }
 
-  def help(): Unit = {
-    outputSupportIO()
+  private def operatorMappingWithScore(): Unit = {
+    val header = Seq("CPUOperator", "Score")
+    println(header.mkString(","))
+    GpuOverrides.execs.values.toSeq.sortBy(_.tag.toString).foreach { rule =>
+      val checks = rule.getChecks
+      if (rule.isVisible && checks.forall(_.shown)) {
+        val cpuName = rule.tag.runtimeClass.getSimpleName
+        val allCols = Seq(cpuName, "2")
+        println(s"${allCols.map(replaceDelimiter(_, ",")).mkString(",")}")
```

Comment:
I don't think we need replaceDelimiter here, the class name should be a single word, same with below